src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -3,16 +3,40 @@
 Release History
 ===============
 
-0.1.6
-+++++
-* Support Autorest 3.0 based SDKs
-
-0.1.5
+1.0.0
 ++++++
-* minor fixes
+* Added support for 2018-03-01 API
+
+ - Job level mounting
+ - Environment variables with secret values
+ - Performance counters settings
+ - Reporting of job specific path segment
+ - Support for subfolders in list files api
+ - Usage and limits reporting
+ - Allow to specify caching type for NFS servers
+ - Support for custom images
+ - Added pyTorch toolkit support
+
+* Added 'job wait' command which allows to wait for the job completion and reports job exit code
+* Added 'usage show' command to list current Batch AI resources usage and limits for different regions
+* National clouds are supported
+* Added job command line arguments to mount filesystems on the job level in addition to config files
+* Added more options to customize clusters - vm priority, subnet, initial nodes count for auto-scale clusters,
+  specifying custom image
+* Added command line option to specify caching type for Batch AI managed NFS
+* Simplified specifying mount filesystem in config files. Now you can omit credentials for Azure File Share and
+  Azure Blob Containers - CLI will populate missing credentials using storage account key provided via command line
+  parameters or specified via environment variable or will query the key from Azure Storage (if the storage account
+  belongs to the current subscription).
+* Job file stream command now auto-completes when the job is completed (succeeded, failed, terminated or deleted).
+* Improved '-o table' support for show operations.
+* Added auto storage option for cluster creation.
```

Comment:
We may have some more recommendations here prior to GA, but this is moving in the right direction and I don't see any reason to block this.

Reply:
thanks